.travis.yml

```diff
@@ -5,12 +5,12 @@ compiler:
   - gcc
 
 before_install:
+  - wget -q -O- https://launchpad.net/ubuntu/+archive/primary/+files/libminizip-dev_1.1-5_amd64.deb | sudo dpkg -i -
```

Comment:
see https://travis-ci.org/XQF/xqf/jobs/67411920
Try something like that:

```
wget -q -O/tmp/libminizip-dev_1.1-5_amd64.deb https://launchpad.net/ubuntu/+archive/primary/+files/libminizip-dev_1.1-5_amd64.deb
sudo dpkg -i /tmp/libminizip-dev_1.1-5_amd64.deb
```

But a more elegant way could be to do _apt repository pinning_, if travis-ci allows it.